contributor/Resource_Sponsor.md

```diff
@@ -0,0 +1,7 @@
+# How to sponsor resources
+We list all of resources that are donated to OpenLab in [website](https://openlabtesting.org/explore/). Now we have many cloud providers to provide VMs for CI testing, and also we have some dedicated resources donated by our sponsors. If you would like to provide your cloud or other kind of resources, read the guide bellow:
+
+1. Open an [issue](https://github.com/theopenlab/openlab/issues/new?template=resource_donating.md&labels=donating) to describe what kind and how many resources you would like to provide, fill out the form to tell the details
+2. After received your donation information, OpenLab governance member @theopenlab/governance will contact you as soon as possible   
+3. The OpenLab governance member will ask operator team @theopenlab/ops to add the cloud provider or other kind of resources to OpenLab CI or dedicated resource pool when donation resources are available
```

Comment:
s/add the cloud provider or other kind of resources/add virtual machine or other kind of resources/